# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <sengedev@gmail.com>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: woeusb\n"
"Report-Msgid-Bugs-To: WoeUSB Issue Tracker <https://github.com/slacka/WoeUSB/"
"issues>\n"
"POT-Creation-Date: 2024-03-30 00:05+0800\n"
"PO-Revision-Date: 2024-03-30 00:09+0800\n"
"Last-Translator: SengeDev <sengedev@gmail.com>\n"
"Language-Team: WoeUSB Issue Tracker <https://github.com/slacka/WoeUSB/"
"issues>\n"
"Language: zh_CN\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-Basepath: ..\n"
"X-Generator: Poedit 3.4.2\n"
"X-Poedit-SearchPath-0: .\n"

#: core.py:83
msgid "You need to specify installation type (--device or --partition)"
msgstr "您需要指定安装类型为设备或分区（--device or --partition）"

#: core.py:148
#, python-brace-format
msgid "Warning: You are not running {0} as root!"
msgstr "警告:您没有以ROOT用户运行{0}！"

#: core.py:149
msgid "Warning: This might be the reason of the following failure."
msgstr "警告： 这可能是导致以下故障的原因。"

#: core.py:163
msgid "Error: Unable to mount source filesystem"
msgstr "错误： 无法挂载源文件系统"

#: core.py:186
msgid "Error: Unable to mount target filesystem"
msgstr "错误： 无法挂载目标文件系统"

#: core.py:222
#, python-brace-format
msgid "Wiping all existing partition table and filesystem signatures in {0}"
msgstr "清除 {0} 中所有现有分区表和文件系统签名"

#: core.py:236
#, python-brace-format
msgid "Ensure that {0} is really wiped..."
msgstr "确保 {0} 真的被擦除..."

#: core.py:243
msgid ""
"Error: Partition is still detected after wiping all signatures, this "
"indicates that the drive might be locked into readonly mode due to end of "
"lifespan."
msgstr ""
"错误： 擦除所有签名后仍检测到分区，这表明硬盘可能因寿命终止而锁定为只读模式。"

#: core.py:257
#, python-brace-format
msgid "Creating new partition table on {0}..."
msgstr "在 {0} 上创建新分区表..."

#: core.py:262
msgid "Error: Currently GUID partition table is not supported."
msgstr "错误： 目前不支持 GUID 分区表。"

#: core.py:265
msgid "Error: Partition table not supported."
msgstr "错误： 不支持的分区表。"

#: core.py:292
msgid "Error: Filesystem not supported"
msgstr "错误： 不支持的文件系统"

#: core.py:295
msgid "Creating target partition..."
msgstr "创建目标分区..."

#: core.py:326
#, python-brace-format
msgid "FATAL: Illegal {0}, please report bug."
msgstr "致命错误：非法的 {0}，请报告错误。"

#: core.py:338
msgid "FATAL: Shouldn't be here"
msgstr "致命错误：不应该在这里"

#: core.py:378
msgid ""
"Warning: Unable to download UEFI:NTFS partition image from GitHub, "
"installation skipped.  Target device might not be bootable if the UEFI "
"firmware doesn't support NTFS filesystem."
msgstr ""
"警告： 无法从 GitHub 下载 UEFI:NTFS 分区映像，安装跳过。 如果 UEFI 固件不支"
"持 NTFS 文件系统，目标设备可能无法启动。"

#: core.py:394
msgid "Mounting source filesystem..."
msgstr "挂载源文件系统..."

#: core.py:399 core.py:434
#, python-brace-format
msgid "Error: Unable to create {0} mountpoint directory"
msgstr "错误： 无法创建 {0} 挂载点目录"

#: core.py:408 core.py:415
msgid "Error: Unable to mount source media"
msgstr "错误： 无法加载源介质"

#: core.py:429
msgid "Mounting target filesystem..."
msgstr "挂载目标文件系统..."

#: core.py:440
msgid "Error: Unable to mount target media"
msgstr "错误： 无法挂载目标媒体"

#: core.py:462
msgid "Copying files from source media..."
msgstr "从源媒体复制文件..."

#: core.py:520
msgid "Installing GRUB bootloader for legacy PC booting support..."
msgstr "安装 GRUB 引导加载程序以支持BIOS/CSM启动..."

#: core.py:541
msgid "Installing custom GRUB config for legacy PC booting..."
msgstr "为BIOS/CSM启动安装自定义 GRUB 配置..."

#: core.py:560
#, python-brace-format
msgid "Unmounting and removing {0}..."
msgstr "卸载并删除 {0}..."

#: core.py:562
msgid "Warning: Unable to unmount filesystem."
msgstr "警告： 无法卸载文件系统。"

#: core.py:568
msgid "Warning: Unable to remove source mountpoint"
msgstr "警告： 无法移除源挂载点"

#: core.py:600 core.py:607
msgid ""
"Some mountpoints are not unmount/cleaned successfully and must be done "
"manually"
msgstr "某些挂载点无法成功卸载/清理，必须手动操作"

#: core.py:605
msgid ""
"We were unable to unmount target filesystem for you, please make sure target "
"filesystem is unmounted before detaching to prevent data corruption"
msgstr ""
"我们无法为您卸载目标文件系统，请在移除前确保目标文件系统已卸载，以防止数据损"
"坏"

#: core.py:612
msgid ""
"Target device is busy, please make sure you unmount all filesystems on "
"target device or shutdown the computer before detaching it."
msgstr ""
"目标设备繁忙，请确保卸载目标设备上的所有文件系统或关闭计算机后再移除目标设"
"备。"

#: core.py:615
msgid "You may now safely detach the target device"
msgstr "现在您可以安全地移除目标设备"

#: core.py:620
msgid "Done :)"
msgstr "完成"

#: core.py:621
msgid "The target device should be bootable now"
msgstr "目标设备现在应该可以启动了"

#: gui.py:34
msgid "Show all drives"
msgstr "显示所有设备"

#: gui.py:35
msgid "Show all drives, even those not detected as USB stick."
msgstr "显示所有设备（即使未被识别为USB设备）。"

#: gui.py:46
msgid "Set boot flag"
msgstr "设置启动标志"

#: gui.py:47
msgid "Sets boot flag after process of copying."
msgstr "在复制过程结束后设置启动标志。"

#: gui.py:49
msgid "Use NTFS"
msgstr "使用NTFS"

#: gui.py:50
msgid "Use NTFS instead of FAT. NOTE: NTFS seems to be slower than FAT."
msgstr "使用 NTFS 而不是 FAT。注意：NTFS 似乎比 FAT 慢。"

#: gui.py:52
msgid "Skip legacy grub bootloader"
msgstr "跳过传统GRUB启动"

#: gui.py:53
msgid ""
"No legacy grub bootloader will be created. NOTE: It will only boot on system "
"with UEFI support."
msgstr ""
"不会创建传统的 grub 引导加载程序。注意：它只能在支持 UEFI 的系统上启动。"

#: gui.py:60
msgid "&File"
msgstr "文件(&F)"

#: gui.py:61
msgid "&Options"
msgstr "选项(&O)"

#: gui.py:62
msgid "&Help"
msgstr "帮助(&H)"

#: gui.py:118
msgid "Source :"
msgstr "源文件:"

#: gui.py:121
msgid "From a disk image (iso)"
msgstr "从一个光盘镜像文件（ISO）"

#: gui.py:126
msgid "Please select a disk image"
msgstr "请选择一个光盘镜像文件"

#: gui.py:132
msgid "From a CD/DVD drive"
msgstr "从CD/DVD设备"

#: gui.py:145
msgid "Target device :"
msgstr "目标设备:"

#: gui.py:157
msgid "Install"
msgstr "安装"

#: gui.py:230
msgid ""
"Are you sure? This will delete all your files and wipe out the selected "
"partition."
msgstr "确定吗？这将删除所有文件并清除所选分区。"

#: gui.py:231 gui.py:264
msgid "Cancel"
msgstr "取消"

#: gui.py:253
msgid "Installing"
msgstr "正在安装"

#: gui.py:253
msgid "Please wait..."
msgstr "请稍等..."

#: gui.py:264
msgid "Are you sure you want to cancel the installation?"
msgstr "您确定要取消安装吗？"

#: gui.py:273
msgid "Installation succeeded!"
msgstr "安装成功！"

#: gui.py:273 gui.py:275
msgid "Installation"
msgstr "安装"

#: gui.py:275
msgid "Installation failed!"
msgstr "安装失败！"

#: gui.py:291
msgid "About"
msgstr "关于"

#: gui.py:322
msgid "Authors"
msgstr "作者"

#: gui.py:325
msgid "Original WinUSB Developer"
msgstr "WinUSB 开发人员"

#: gui.py:327
msgid ""
"\n"
"            This file is part of WoeUSB-ng.\n"
"\n"
"            WoeUSB-ng is free software: you can redistribute it and/or "
"modify\n"
"            it under the terms of the GNU General Public License as "
"published by\n"
"            the Free Software Foundation, either version 3 of the License, "
"or\n"
"            (at your option) any later version.\n"
"\n"
"            WoeUSB-ng is distributed in the hope that it will be useful,\n"
"            but WITHOUT ANY WARRANTY; without even the implied warranty of\n"
"            MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n"
"            GNU General Public License for more details.\n"
"\n"
"            You should have received a copy of the GNU General Public "
"License\n"
"            along with WoeUSB-ng.  If not, see <http://www.gnu.org/licenses/"
">.\n"
"        "
msgstr ""
"\n"
"            该文件是 WoeUSB-ng 的一部分。\n"
"\n"
"            WoeUSB-ng 是自由软件：您可以根据 GNU General Public License（GNU "
"通用公共许可证）发布的条款重新发布和/或修改它。\n"
"            或对其进行修改。\n"
"            自由软件基金会发布的 GNU 通用公共许可证的条款下重新发布和/或修改"
"它。\n"
"            (由您选择）任何后续版本。\n"
"\n"
"            WoeUSB-ng 的发布是希望它能派上用场、\n"
"            但不附带任何保证；甚至不附带以下默示保证\n"
"            适销性或特定用途的适用性。 参见\n"
"            更多详情，请参阅 GNU 通用公共许可证。\n"
"\n"
"            您应该随 WoeUSB-nguer 一起收到一份 GNU 通用公共许可证的副本。\n"
"            的副本。 如果没有，请参阅 <http://www.gnu.org/licenses/>。\n"
"        "

#: gui.py:347
msgid "License"
msgstr "许可证"

#: utils.py:38
#, python-brace-format
msgid ""
"Error: {0} requires {1} command in the executable search path, but it is not "
"found."
msgstr "错误： {0}需要可执行搜索路径中的{1}命令，但未找到该命令。"

#: utils.py:50
msgid "Error: mkdosfs/mkfs.msdos/mkfs.vfat/mkfs.fat command not found!"
msgstr "错误：未找到 mkdosfs/mkfs.msdos/mkfs.vfat/mkfs.fat 命令！"

#: utils.py:51
msgid "Error: Please make sure that dosfstools is properly installed!"
msgstr "错误： 请确保 dosfstools 已正确安装！"

#: utils.py:56
msgid "Error: mkntfs command not found!"
msgstr "错误：未找到 mkntfs 命令！"

#: utils.py:57
msgid "Error: Please make sure that ntfs-3g is properly installed!"
msgstr "错误： 请确保已正确安装 ntfs-3g！"

#: utils.py:67
msgid "Error: grub-install or grub2-install command not found!"
msgstr "错误：未找到 grub-install 或 grub2-install 命令！"

#: utils.py:68
msgid "Error: Please make sure that GNU GRUB is properly installed!"
msgstr "错误： 请确保 GNU GRUB 已正确安装！"

#: utils.py:86
#, python-brace-format
msgid ""
"Error: Source media \"{0}\" not found or not a regular file or a block "
"device file!"
msgstr "错误： 未找到源媒体\"{0}\"，或不是常规文件或块设备文件！"

#: utils.py:92
#, python-brace-format
msgid "Error: Target media \"{0}\" is not a block device file!"
msgstr "错误： 目标媒体\"{0}\"不是块设备文件！"

#: utils.py:96
#, python-brace-format
msgid "Error: Target media \"{0}\" is not an entire storage device!"
msgstr "错误： 目标媒体\"{0}\"不是整个存储设备！"

#: utils.py:100
#, python-brace-format
msgid "Error: Target media \"{0}\" is not an partition!"
msgstr "错误： 目标媒体\"{0}\"不是分区！"

#: utils.py:122
#, python-brace-format
msgid "Info: Target device is {0}"
msgstr "提示：目标设备是 {0}"

#: utils.py:123
#, python-brace-format
msgid "Info: Target partition is {0}"
msgstr "提示：目标分区是 {0}"

#: utils.py:136
#, python-brace-format
msgid "Warning: The following partitions will be unmounted: {0}"
msgstr "警告： 以下分区将被卸载： {0}"

#: utils.py:152
msgid ""
"Error: Source media is currently mounted, unmount the partition then try "
"again"
msgstr "错误： 源介质当前已加载，请卸载分区，然后重试"

#: utils.py:157
msgid ""
"Error: Target partition is currently mounted, unmount the partition then try "
"again"
msgstr "错误： 目标分区当前已挂载，请卸载分区，然后重试"

#: utils.py:164
msgid ""
"Error: Target device is currently busy, unmount all mounted partitions in "
"target device then try again"
msgstr "错误： 目标设备当前繁忙，请卸载目标设备中的所有已挂载分区，然后重试"

#: utils.py:180
#, python-brace-format
msgid ""
"Warning: File {0} in source image has exceed the FAT32 Filesystem 4GiB "
"Single File Size Limitation, swiching to NTFS filesystem."
msgstr ""
"警告： 源镜像中的文件 {0} 已超出 FAT32 文件系统 4GiB 单个文件大小限制，请切换"
"到 NTFS 文件系统。"

#: utils.py:185
msgid ""
"Refer: https://github.com/slacka/WoeUSB/wiki/Limitations#fat32-"
"filesystem-4gib-single-file-size-limitation for more info."
msgstr ""
"更多信息请参阅：https://github.com/slacka/WoeUSB/wiki/Limitations#fat32-"
"filesystem-4gib-single-file-size-limitation。"

#: utils.py:209
msgid ""
"Error: Target filesystem not supported, currently supported filesystem: FAT, "
"NTFS."
msgstr "错误： 不支持目标文件系统，当前支持文件系统： FAT、NTFS。"

#: utils.py:230
msgid ""
"Warning: Your device doesn't seems to have an UEFI:NTFS partition, UEFI "
"booting will fail if the motherboard firmware itself doesn't support NTFS "
"filesystem!"
msgstr ""
"警告： 您的设备似乎没有 UEFI:NTFS 分区，如果主板固件本身不支持 NTFS 文件系"
"统，UEFI 启动将失败！"

#: utils.py:233
msgid ""
"Info: You may recreate disk with an UEFI:NTFS partition by using the --"
"device creation method"
msgstr "提示：您可以使用--设备创建方法，重新创建带有 UEFI:NTFS 分区的磁盘"

#: utils.py:265
msgid "Error: Not enough free space on target partition!"
msgstr "错误： 目标分区可用空间不足！"

#: utils.py:267
#, python-brace-format
msgid "Error: We required {0}({1} bytes) but '{2}' only has {3}({4} bytes)."
msgstr "错误： 我们需要 {0}({1} 字节)，但'{2}' 只有 {3}({4} 字节)。"

#: workaround.py:16
msgid "Making system realize that partition table has changed..."
msgstr "让系统意识到分区表已更改..."

#: workaround.py:19
msgid "Wait 3 seconds for block device nodes to populate..."
msgstr "等待区块设备节点填充 3 秒钟..."

#: workaround.py:35
msgid ""
"Applying workaround for buggy motherboards that will ignore disks with no "
"partitions with the boot flag toggled"
msgstr ""
"正在应用变通方法解决主板上的漏洞，这些漏洞会在切换启动标志时忽略没有分区的磁"
"盘"

#: workaround.py:62
msgid ""
"Source media seems to be Windows 7-based with EFI support, applying "
"workaround to make it support UEFI booting"
msgstr "源介质似乎基于 Windows 7，支持 EFI，正在使用变通方法使其支持 UEFI 启动"

#: workaround.py:71
#, python-brace-format
msgid "DEBUG: Can't find efi directory, use {0}"
msgstr "调试： 找不到 efi 目录，使用 {0}"

#: workaround.py:75 workaround.py:87
#, python-brace-format
msgid "DEBUG: {0} detected."
msgstr "调试: 检测到 {0}。"

#: workaround.py:83
#, python-brace-format
msgid "DEBUG: Can't find efi/boot directory, use {0}"
msgstr "调试： 找不到 efi/boot 目录，使用 {0}"

#: workaround.py:95
msgid "INFO: Detected existing EFI bootloader, workaround skipped."
msgstr "提示： 检测到现有 EFI 引导加载程序，跳过解决方法。"
